backend/src/models/expense.ts

```diff
@@ -0,0 +1,10 @@
+export type Expense = {
+  expense_id: number;
+  category_id: number;
+  amount: number;
+  Expense_date: Date;
+  summary: string | null;
+};
+
+export type CreateExpensePayload = Omit<Expense, 'expense_id' | 'category_id'>;
+export type BaseExpense = Omit<Expense, 'expense_id'>;
```

Comment:
Base entity is the entity without the primary id field since the db will automatically populate it